libopenage/gui/assetmanager_link.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright 2015-2016 the openage authors. See copying.md for legal info.
+
+#include "assetmanager_link.h"
+
+#include <QtQml>
+
+namespace openage {
+namespace gui {
+
+namespace {
+const int registration = qmlRegisterType<AssetManagerLink>("com.openage", 1, 0, "AssetManager");
```

Comment:
Why `com.openage`? Can't we use something less corporate like `lol.openage`? :grinning: